src/Sylius/Bundle/ShopBundle/templates/shared/order/show/summary.html.twig

```diff
@@ -0,0 +1 @@
+{% hook 'summary' %}
```

Comment:
I have some doubts about these shared templates and hooks, but if I see it correctly (correct me if I'm wrong), it works as it should, so that they release hooks for both shared and account context? I'm asking, because in the end application, you should be able to change something either everywhere or in a specific place. How does that look at the moment? What hook is being triggered for example here?

Reply:
Yes, correct. In this case both hooks are released and it can be modified globally for shared component or for specific case where it is called. Although i checked other places and there was a case where it did not work like this(shared/address.html.twig), so i fixed it.